src/actions/preferences.js

```diff
@@ -13,12 +13,12 @@ import {getPreferenceKey} from 'utils/preference_utils';
 import {bindClientFunc, forceLogoutIfNecessary} from './helpers';
 import {getLogErrorAction} from './errors';
 
-export function deletePreferences(preferences) {
+export function deletePreferences(userId, preferences) {
```

Comment:
dont know about passing the userId as you should have it in the preferences, right?